mlir/include/mlir/IR/Block.h

```diff
@@ -215,7 +215,7 @@ class Block : public IRObjectWithUseList<BlockOperand>,
   Operation *getTerminator();
 
   /// Check whether this block has a terminator.
```

Comment:
Nit: make sure the comment reflects the new function logic.

Reply:
Nice catch. Also updated the comments in `getTerminator`